README.md

```diff
@@ -233,3 +237,10 @@ or IntelⓇ FPGA RTE for OpenCL™ installed on your system:
     ```
 
 6.  Run your OpenCL host program.
+
+### Notes
```

Comment:
Added a note section because this seems to apply to both with or without IntelⓇ FPGA SDK for OpenCL™ or IntelⓇ FPGA RTE for OpenCL™?